### PR TITLE
Offer async command for calling the PAF library

### PR DESCRIPTION
**Pull request by RomainLofaso**

### CODE REVIEW

**Review by Aerion — APPROVED** [review]
LGTM, though I'd suggest waiting for @OlivierChirouze feedback as there was something blocking on the closed PR(?)

File: paf-mvp-frontend/src/main.ts [hunk 1]
```diff
@@ -19,4 +24,17 @@ const showNotification = (type: NotificationEnum) => notificationService.showNot
 
 // TODO: avoid global declaration
 window.PAFUI ??= { promptConsent, showNotification };
-window.PAF ??= { getNewId, signPreferences, getIdsAndPreferences, refreshIdsAndPreferences, updateIdsAndPreferences };
+window.PAF = {
+  // If the queue has been defined outside of this script, will be replaced by it
```
Comment (OlivierChirouze) [inline paf-mvp-frontend/src/main.ts:28]: NIT: this comment is not relevant anymore

File: paf-mvp-frontend/src/utils/queue.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+//  Call of the PAF-lib when loaded asynchronously:
+// setUpImmediateProcessingQueue(window.PAF);
+
+/** An operation executed asynchronously after the PAF-lib is loaded. */
+export type Command = () => void;
+
+/**
+ * Queue for processing pushed commands.
+ *
+ * Note: An Array<Operation> is considered
```
Comment (OlivierChirouze) [inline paf-mvp-frontend/src/utils/queue.ts:22]: NIT I guess it's `Array<Command>`

**Review by OlivierChirouze — APPROVED** [review]
👍

File: paf-mvp-frontend/src/utils/queue.ts (hunk 2 above)
Comment (RomainLofaso) [reply paf-mvp-frontend/src/utils/queue.ts]: Fixed

File: paf-mvp-frontend/src/main.ts (hunk 1 above)
Comment (RomainLofaso) [reply paf-mvp-frontend/src/main.ts]: Fixed

**Pull request merged**